utils/github.go

```diff
@@ -356,3 +378,46 @@ func GetReviewThreads(ctx context.Context, client *githubv4.Client, owner string
 
 	return reviewThreads, nil
 }
+
+func GetPullRequestLastPushedEventDate(ctx context.Context, client *githubv4.Client, owner string, repo string, number int) (time.Time, error) {
+	var lastPushedEventDateQuery LastPushedEventDateQuery
+	varGQLastPushedEventDate := map[string]interface{}{
+		"repositoryOwner":   githubv4.String(owner),
+		"repositoryName":    githubv4.String(repo),
+		"pullRequestNumber": githubv4.Int(number),
+	}
+
+	err := client.Query(ctx, &lastPushedEventDateQuery, varGQLastPushedEventDate)
+	if err != nil {
+		return time.Time{}, err
+	}
+	if len(lastPushedEventDateQuery.Repository.PullRequest.TimelineItems.Nodes) == 0 {
+		return time.Time{}, errors.New("pushedDate not found")
+	}
```

Comment:
Let's make more verbose here and also add some lines between.

```suggestion
	
	hasLastPush := len(lastPushQuery.Repository.PullRequest.TimelineItems.Nodes) > 0
	if  !hasLastPush {
		return time.Time{}, errors.New("last push not found")
	}
	
```